sotodlib/preprocess/processes.py

```diff
@@ -1355,6 +1357,28 @@ def save(self, proc_aman, split_flg_aman):
         if self.save_cfgs:
             proc_aman.wrap("split_flags", split_flg_aman)
 
+class UnionFlags(_Preprocess):
+    """Do the union of relevant flags for mapping
+    Typically you would include turnarounds, glitches, etc.
+
+    Saves results for aman under the "flags.glitch_flags" field.
+
+     Example config block::
+
+        - name : "union_flags"
+          process:
+            flag_labels: ['jumps_2pi.jump_flag', 'glitches.glitch_flags', 'turnaround_flags.turnarounds']
+
+    """
+    name = "union_flags"
+    
+    def process(self, aman, proc_aman):
+        total_flags = so3g.proj.RangesMatrix.zeros([proc_aman.dets.count, proc_aman.samps.count]) # get an empty flags with shape (Ndets,Nsamps)
+        for label in self.process_cfgs['flag_labels']:
+            _label = attrgetter(label)
+            total_flags += _label(proc_aman) # The + operator is the union operator in this case
+        aman['flags'].wrap('glitch_flags', total_flags)
```

Comment:
Should the union of the jump, glitch, and turnarounds be called glitch_flags, or should they maybe be called something more general (maybe just `aman.flags['flags']` or something like that) since all of those are not really related to glitches?  Also, if the glitch flag axis manager is merged in `get_glitch_flags`, there can be a potential name conflict.

Also, it might be useful to have a check whether a flag axis manager exists and maybe a clean error message if it doesn't.

Reply:
This is the ACT legacy. In ACT the overall flags are called glitch_flags. We should do a more general name and we should discuss it, but for now, we have the ACT convention all over sotodlib.

Reply:
I agree with both of Michael's points, fine to push off the change away from the `glitch_flags` naming convention from ACT to another PR but please open an issue for it.

I also agree that there needs to be some logic here for handling `aman.flags` if it doesn't exist you need to create it, and if it exists but `glitch_flags` is already populated you need to first delete what's in that axis before writing to it.

Reply:
Ok I made a new PR